### PR TITLE
Qa auto sql

### PR DESCRIPTION
**Pull request by Rulesofmadnes**
updated tables in sql

### CODE REVIEW

File: qa_auto_table.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    lastName VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL
+);
+CREATE TABLE cars (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    userId INT NOT NULL,
+    carBrandId INT NOT NULL,
+    carModelId INT NOT NULL,
+    mileage INT,
```
Comment (Settt) [inline qa_auto_table.sql]: float

File: qa_auto_table.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    initialMilleage INT,
```
Comment (Settt) [inline qa_auto_table.sql]: float

File: updated_qa_auto.sql [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    lastName VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL
+);
+CREATE TABLE cars (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    userId INT,
+    carBrandId INT,
+    carModelId INT,
+    mileage INT,
```
Comment (Settt) [inline updated_qa_auto.sql]: float

File: updated_qa_auto.sql [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    initialMileage INT,
```
Comment (Settt) [inline updated_qa_auto.sql]: float

**Review by Settt — CHANGES_REQUESTED** [review]

File: updated_qa_auto.sql (hunk 3 above)
Comment (Rulesofmadnes) [reply updated_qa_auto.sql]: corrected

File: updated_qa_auto.sql (hunk 4 above)
Comment (Rulesofmadnes) [reply updated_qa_auto.sql]: corrected

**Pull request merged**